db/migrate/20150629180552_change_password_type_user.rb

```diff
@@ -0,0 +1,6 @@
+class ChangePasswordTypeUser < ActiveRecord::Migration
+  def change
+    remove_column :users, :password
+    add_column :users, :password, :varchar, null: false
```

Comment:
yeah, In this migration earlier it was intended to store it as `password_hash` and column type was changed to `blob`. but then, I  migrated to postgresql with varchar didn't realise the name was exactly the same.

For migration of existing users, add `password_hash` column, hash the existing user's password. drop the `password` column and use the password_hash column henceforth.